peacecorps/peacecorps/static/peacecorps/js/src/showOnce.js

```
@@ -0,0 +1,23 @@
+/**
+ * Content that should only be shown once per user. A flag is set in
+ * localstorage, and, if present, the content is hidden.
+ **/
+'use strict';
+
+var $ = require('jquery');
+
+var showOnce = function($root, storage) {
+  $root.find('[data-show-once]').each(function() {
+    var $el = $(this),
+        key = 'show-once:' + $el.attr('data-show-once'),
+        seen = !!(storage && storage.getItem(key));
```

Seen is false and it proceeds to show the beta warning. Originally it'd silently fail, but testing pointed out the issue.